schema/transportation/segment.yaml

```diff
@@ -308,6 +309,30 @@ properties:
         - dirt
         - paving_stones
         - metal
+    routes:
```

Comment:
This schema proposal does allow multiple road numbers for a single road: the `routes` property is an array but I was wondering if there is a possibility to indicate what the most relevant route number is. 
In Europe, roads typically have multiple route numbers (international, national, regional) and in public OSM there's a capability to indicate the most relevant one (e.g. international), see this example road in Belgium: https://www.openstreetmap.org/way/582460822
![image](https://github.com/OvertureMaps/schema/assets/73464443/29741f68-2738-42a0-82b4-999156b8ee86)

Reply:
Great point, the goal with this initial schema is just to provide the "raw" data for consumption as-is from OSM. I do think this should be part of the *next* iteration where we further process + schematize, adding properties like `network_country`, `network_is_intl`, etc as discussed in the task force.

I didn't want to tackle that additional schematizing because of the wide array of tagging patterns that need to be deeply understood. US highways for example have a similar intra vs interstate network hierarchy but that doesn't always come from dedicated tags although those do exist (`is_in:state`). Typically these are just value patterns - "US:I" or "US:MA" for example.

Reply:
Could we indicate a priority by saying the first entry is the most relevant?

Reply:
What would we use to determine priority or relevance? I could definitely add a "best effort" ordering by international, national, regional, etc using some of these known tagging patterns without too much effort.

I still think the right way is to get a deeper understanding and have future schema improvements to capture this. Any priority we choose could be wrong depending on context